### DISCUSSION TITLE
Variabler Strompreis

### DISCUSSION THREAD

**Discussion by bscheufele** [solectrus/solectrus#2493] [Ideas]
Tolles Projekt!

Für die Berechnung der Einsparung wäre es cool den Strompreis variable zu gestalten. Also Strompreis pro Jahr oder so ähnlich. Dann wird die Einsparung noch genauer berechnet.

**Comment by ledermann** [comment]
Verstehe ich nicht so ganz, der Strompreis **ist** doch bereits variabel. In den Einstellungen kann erfasst werden, wie sich der Strompreis im Zeitverlauf ändert, hier ein Beispiel:
https://solectrus.ledermann.dev/settings/prices?name=electricity

Bei der Berechnung der Ersparnis wird das tagesgenau berücksichtigt.

Was derzeitig tatsächlich **nicht** unterstützt wird, sind dynamische Stromtarife wie z.B. Tibber, deren Preis sich täglich ändert - siehe #2150

**Comment by bscheufele** [comment]
Das hab ich nicht gewusst - cool dann ist das ja schon drin 🤩

**Reply by ledermann** [reply]
Wobei das übrigens auch ein großer Vorteil gegenüber der SENEC-App ist, die ja nur **einen einzigen** Strompreis kennt.
